python/cudf/benchmarks/common/utils.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+"""Common utilities for fixture creation and benchmarking."""
+
+import inspect
+import re
+import textwrap
+from collections.abc import MutableSet
+from itertools import groupby
+from numbers import Real
+
+import pytest_cases
+from config import NUM_COLS, NUM_ROWS, cudf, cupy
+
+
+def make_gather_map(len_gather_map: Real, len_column: Real, how: str):
+    """Create a gather map based on "how" you'd like to gather from input.
+    - sequence: gather the first `len_gather_map` rows, the first thread
+                collects the first element
+    - reverse:  gather the last `len_gather_map` rows, the first thread
+                collects the last element
+    - random:   create a pseudorandom gather map
+
+    `len_gather_map`, `len_column` gets rounded to integer.
```

Comment:
Why rounding instead of truncation, or requiring that inputs are `int` and not `Real`?

Reply:
That's a question for @isVoid.

Reply:
I think requiring inputs are `int` is the way to go here. @vyasr I don't think you should address this in this PR. I'll make an issue after you merge this.